helm/cluster-autoscaler-app/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v1
 appVersion: v1.24.0
 description: A Helm chart for the cluster autoscaler.
+icon: https://s.giantswarm.io/app-icons/k8s-initiator/1/dark.svg
```

Comment:
k8s-initiator icon for cluster-autoscaler?

Reply:
I understand your confusion. We are reusing the existing icons. K8s-initiator is just the Kubernetes icon. 
As far as I understand it, autoscaler is from Kubernetes, right?

If the Kubernetes icon does not fit, I am open to suggestions.

Reply:
works for me